lapis/src/main/kotlin/org/genspectrum/lapis/model/mutationsOverTime/MutationsOverTimeModel.kt

```diff
@@ -14,65 +16,91 @@ import org.genspectrum.lapis.silo.SiloAction
 import org.genspectrum.lapis.silo.SiloClient
 import org.genspectrum.lapis.silo.SiloFilterExpression
 import org.genspectrum.lapis.silo.SiloQuery
+import org.genspectrum.lapis.silo.WithDataVersion
 import org.springframework.stereotype.Component
 import java.time.LocalDate
 
-data class MutationOverTimeResponse(
-    var rowLabels: List<NucleotideMutation>,
+data class MutationsOverTimeResult(
+    var rowLabels: List<String>,
     var columnLabels: List<DateRange>,
-    var data: List<List<MutationOverTimeCell>>,
+    var data: List<List<MutationsOverTimeCell>>,
```

Comment:
```suggestion
    @param:Schema(
        description = "A 2D array of result cells. " +
            "It should be understood as data[index of mutation][index of date range].",
    )
    var data: List<List<MutationsOverTimeCell>>,
```

(I'm quite sure the text can be improved)